README.md

```diff
@@ -139,6 +139,11 @@ cm run script \
 
 [Apache 2.0](LICENSE.md)
 
+## CM concepts
+
+* https://doi.org/10.5281/zenodo.8105339
+* https://arxiv.org/abs/2406.16791
```

Comment:
We should remove this arxiv paper - it has many factually wrong things. You had earlier agreed to fix them.

Reply:
? 
It has a few minor typos that I still need to clean up but it was not urgent. But I you feel that something is wrong, I will remove it for now to discuss it later. There are also many other things I would like to clarify with the very recent CM/CM4MLOps developments while I was on a sick leave before updating the paper.

Reply:
Here is the PR: https://github.com/mlcommons/cm4mlops/pull/336

Reply:
Thank you Grigori. Yes, not just the typos - "Arjun Suresh - a senior software engineer from cKnowledge.org" - I was at OctoML and only later at cKnowledge and even there the title was not the same. And I think in the last section "Future Plans" you made a typo to use "We" as everywhere else in the PDF it is "I".